Sources/XcodeProj/Objects/Files/PBXGroup.swift

```diff
@@ -233,3 +242,24 @@ public extension PBXGroup {
         return fileReference
     }
 }
+
+private extension PBXGroup {
+
+    func existingFileReference(
+        from objects: PBXObjects? = nil,
```

Comment:
Since the original already accessed a `PBXObjects`, I thought to at least keep that as an optional arg. It seems to be just a constant reference, but playing it safe here: https://github.com/tuist/XcodeProj/blob/6943d5c5f9085d070dbde3c669144e30d41ec635/Sources/XcodeProj/Objects/Project/PBXObject.swift#L75-L81